controllers/storageclaim_controller.go

```diff
@@ -499,6 +504,10 @@ func (r *StorageClaimReconciler) getCephRBDStorageClass(data map[string]string)
 		Provisioner:          csi.GetRBDDriverName(),
 		Parameters:           data,
 	}
+
+	if isEncryptionRequired {
+		storageClass.ObjectMeta.Annotations["keyrotation.csiaddons.openshift.io/schedule"] = "@weekly"
```

Comment:
this should be set only if https://github.com/red-hat-storage/ocs-client-operator/blob/9dea589a7a1f6cbeff1b2ddceb3ceae8e24037f3/api/v1alpha1/storageclaim_types.go#L51 is set.